src/server/handlers/webhook.js

```diff
@@ -0,0 +1,9 @@
+const logging = require('../logging/').default;
+const logger = logging.getLogger('express');
+
+export const notify = (req, res) => {
+  // Acknowledge webhook
+  res.status(200).send();
```

Comment:
Yeah, just logging it for now is fine, and I agree we ought to retain logging even once full integration is in place.  The webhook will basically need to do `lp.snaps.findByURL` on the repository URL (with memcache) and then `snap.requestAutoBuilds` on that, but I'm still putting together the pieces of the LP interaction here.